packages/cli/src/blueprints/package.json

```diff
@@ -29,7 +29,7 @@
     "lint:types": "tsc --noEmit",
     "prepare": "pnpm build",
     "release:changelog": "changeset version",
-    "release:publish": "pnpm build && changeset publish",
+    "release:publish": "changeset publish",
```

Comment:
I found out that, when `pnpm build && pnpm publish` is run, the `prepare` script is run before `publish`. As a result, the `build` script is run twice when `release:publish` is called.

I'm not sure yet if the same conclusion can be made when `changeset publish` is used instead of `pnpm publish`.

Reply:
It turned out, because `changset` publishes the package(s), the `prepare` script does not run before `publish`. So `pnpm build && changeset publish` is correct.